js/build.js

```diff
@@ -19,13 +19,15 @@
   });
 })();
 
-Fliplet.Widget.instance('list', function (data) {
+Fliplet.Widget.instance('list', function(data) {
   var $container = $(this);
 
+  $container.translate({ swipeToSaveLabel: data.swipeToSaveLabel || 'My list' });
```

Comment:
I decided to pass the variable in the parameters of the translation function call. So far, this is the most optimal option for passing the value of a variable. Can you advice some alternatives on how to do this? Using an options from i18n with additional data attribute in html like data-translate-options will be better but we did not find a support for this.

Reply:
Do I need to translate the default "My List" title

Reply:
@skrupnyk Yes, the "My List" title should be translated.

Our JS API suggests that `data-translate-options` should work as suggested in https://github.com/i18next/jquery-i18next#using-options-in-translation-function since we mapped it over to `data-translate-options`.

![image](https://user-images.githubusercontent.com/290733/136711761-ddde14e2-0cb4-4ce0-968d-c710d72aa24d.png)

Please have a go and let us know if it doesn't work. Though, I think it's better to do it in the way you've done it right now. Why do you think it's better to use `data-translate-options`?

Reply:
![image](https://user-images.githubusercontent.com/84472843/136765330-0003969e-0080-4cdf-8308-c16a5e3986f9.png)
![image](https://user-images.githubusercontent.com/84472843/136765459-d3eb1254-402d-487d-9bb2-eb6cae96626d.png)

such an implementation does not work

Reply:
Do we need to update CLI or anything to warm it up? We gave it a quick shot and it still doesn't catch our value :(

Reply:
@upplabs-alex-levchenko Looks like from my call with @skrupnyk he managed to get it working. Though, he was trying to use a variable, the options attribute only supports static string values.

I recommended using another piece of script to add the `data-translate-options` attribute first before executing `$.fn.translate()`.

Have they been able to figure that out?